hack/clean-e2e.sh

```diff
@@ -6,9 +6,11 @@ cd "${REPO_ROOT}" || exit 1
 minikube delete
 docker stop vbmc
 docker rm vbmc
+docker rm -f image-server-e2e
```

Comment:
Nit: Let's keep these consistent
```suggestion
docker rm -f vbmc
docker rm -f image-server-e2e
```